core-bundle/src/Resources/contao/widgets/FileSelector.php

```diff
@@ -0,0 +1,670 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao;
+
+use Contao\Image\ResizeConfiguration;
+use Doctrine\DBAL\Exception\DriverException;
+use Symfony\Component\Filesystem\Path;
+use Symfony\Component\HttpFoundation\Session\Attribute\AttributeBagInterface;
+
+trigger_deprecation('contao/core-bundle', '4.13', 'Using the "Contao\FileSelector" class has been deprecated and will no longer work in Contao 5.0. Use the picker instead.');
+
+/**
+ * Provide methods to handle input field "file tree".
+ *
+ * @property string  $path
+ * @property string  $fieldType
+ * @property string  $sort
+ * @property boolean $files
+ * @property boolean $filesOnly
+ * @property string  $extensions
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ *
+ * @deprecated Deprecated since Contao 4.13, to be removed in Contao 5.0.
+ *             Use the picker instead.
+ */
+class FileSelector extends Widget
```

Comment:
Did you accidentally add this class again?

Reply:
Yes 🙈

Reply:
Fixed in b06a6f671e9a0fe11d079540dfb232b67f6d64ae